server/src/handlers/posts/create-post.ts

```diff
@@ -1,19 +1,33 @@
 import { Request, Response } from 'express'
 import { Db } from 'mongodb'
+import { log } from 'util'
 
 /** creates a new post, returns post id */
 export const createPostHandler = (database: Db) => async (
   req: Request,
   res: Response,
 ) => {
-  const { post } = req.body
+  try {
+    // TODO:
+    // 1. create a new post for user
+    // 2. get created post id
+    // 3. return post id
 
-  // TODO:
-  // 1. create a new post for user
-  // 2. get created post id
-  // 3. return post id
+    const { title, description } = req.body
 
-  return res.status(404).json({
-    message: 'not implemented',
-  })
+    const createPost = await database.collection('posts').insertOne({
+      title,
+      description,
+      data: Date.now()
+    })
+    
+    return res.status(200).json({
+      postID: createPost.insertedId
+    })
+
+  } catch (error) {
+    return res.status(404).json({
+      message: 'not implemented',
```

Comment:
Misleading status and messagee

Reply:
Do you mean I haven't logged an error in catch block?

Reply:
I mean 404 is wrong code and 'not implemented' is a bad message